cli/src/config.rs

```diff
@@ -69,10 +69,10 @@ pub struct Configuration {
 #[config(env_prefix = "IROHA_NETWORK_")]
 pub struct NetworkConfiguration {
     /// Actor mailbox size
-    pub mailbox: usize,
```

Comment:
there is also the possibility to use serde field attributes to make custom serializations. With field attributes you can have a field on a struct be defined as `usize` and have it consistently serialized as `u32`(or `u64`). This might make sense if you will be converting `u32` into `usize` at the point of use anyway. @appetrosyan thoughts?

Reply:
I think that making things fixed, portable and straightforward is preferable. We have plenty of structures that need a custom serialiser.